### PR TITLE
Implement L10n in cross signing

### PR DESCRIPTION
**Pull request by bitfriend**
Replace the raw text with l10n in UI of cross signing

### CODE REVIEW

File: app/lib/l10n/app_en.arb [hunk 1]
```diff
@@ -222,7 +222,16 @@
   "remove": "Remove",
   "reset": "Reset",
   "sasGotIt": "Got it",
-  "sasIncomingReqNotifContent": "%s wants to verify your session",
+  "sasIncomingReqNotifContent": "{sender} wants to verify your session",
+  "@sasIncomingReqNotifContent": {
+    "description": "A message with a single parameter",
```
Comment (gnunicorn) [inline app/lib/l10n/app_en.arb:227]: this is a bad description.
```suggestion
    "description": "When another users asks to verify this sessions",
```

File: app/lib/l10n/app_en.arb [hunk 2]
```diff
@@ -278,12 +287,31 @@
   "verificationCodeNotice": "Compare the code with the one displayed on the other user’s screen.",
   "verificationConclusionCompromised": "One of the following may be compromised:\n\n   - Your homeserver\n   - The homeserver the user you’re verifying is connected to\n   - Yours, or the other users’ internet connection\n   - Yours, or the other users’ device",
   "verificationConclusionNotSecure": "Not secure",
-  "verificationConclusionOkDone": "You’ve successfully verified %s!",
+  "verificationConclusionOkDone": "You’ve successfully verified {sender}!",
+  "@verificationConclusionOkDone": {
+    "description": "A message with a single parameter",
```
Comment (gnunicorn) [inline app/lib/l10n/app_en.arb:292]:
```suggestion
    "description": "After successfully verifying another users",
```

File: app/lib/l10n/app_en.arb [hunk 3]
```diff
@@ -278,12 +287,31 @@
...
+  },
   "verificationConclusionOkNotice": "Messages with this user are end-to-end encrypted and can’t be read by third parties.",
   "verificationConclusionOkSelfNotice": "Your new session is now verified. It has access to your encrypted messages, and other users will see it as trusted.",
   "verificationEmojiNotice": "Compare the unique emoji, ensuring they appear in the same order.",
   "verificationNoScanEmojiTitle": "Verify by comparing emojis",
-  "verificationRequestWaitingFor": "Waiting for %s…",
+  "verificationRequestAccept": "To proceed, please accept the verification request on your other device.",
+  "verificationRequestWaitingFor": "Waiting for {sender}…",
+  "@verificationRequestWaitingFor": {
+    "description": "A message with a single parameter",
```
Comment (gnunicorn) [inline app/lib/l10n/app_en.arb:307]:
```suggestion
```

**Review by gnunicorn — APPROVED** [review]
Let's make sure to keep the helpers in the text in high quality, too.

File: app/lib/features/cross_signing/views/request_ready.dart [hunk 4]
```diff
@@ -49,23 +48,25 @@ class RequestReadyView extends StatelessWidget {
...
+          //       const Padding(
           //         padding: EdgeInsets.all(8),
           //         child: Icon(Atlas.camera),
           //       ),
-          //       Text('Scan with this device'),
+          //       Text(L10n.of(context).verificationScanWithThisDevice),
           //     ],
           //   ),
           // ),
           // const Spacer(),
```
Comment (kumarpalsinh25) [inline app/lib/features/cross_signing/views/request_ready.dart:62]: Let's remove commented code if not in use.

File: app/lib/features/cross_signing/views/sas_cancelled.dart [hunk 5]
```diff
@@ -43,7 +43,7 @@ class SasCancelledView extends StatelessWidget {
           SizedBox(
             width: MediaQuery.of(context).size.width * 0.40,
             child: ElevatedButton(
-              child: const Text('Got it'),
+              child: Text(L10n.of(context).sasGotIt),
```
Comment (kumarpalsinh25) [inline app/lib/features/cross_signing/views/sas_cancelled.dart:46]: Prefix `sas` stands for?

**Review by kumarpalsinh25 — APPROVED** [review]
LGTM!

File: app/lib/features/cross_signing/views/request_ready.dart (hunk 4 above)
Comment (bitfriend) [reply app/lib/features/cross_signing/views/request_ready.dart:62]: Besides emoji, QR code scanning is another way to perform cross signing.
I left QR code for future implementation.

File: app/lib/features/cross_signing/views/sas_cancelled.dart (hunk 5 above)
Comment (bitfriend) [reply app/lib/features/cross_signing/views/sas_cancelled.dart:46]: `sas` stands for short authentication string.
Event handling of verification consists of request stage and sas stage.
Request stage stands for stage prior to request accept.
Sas stage stands for stage after request accepted.
There are many sentences related to verification, and I split them into request group and sas group.

File: app/lib/l10n/app_en.arb (hunk 1 above)
Comment (bitfriend) [reply app/lib/l10n/app_en.arb]: Fixed by https://github.com/acterglobal/a3/pull/1539/commits/0a6fcae7725f68ece46f80a92bcfccf852c8f2c0

File: app/lib/l10n/app_en.arb (hunk 2 above)
Comment (bitfriend) [reply app/lib/l10n/app_en.arb]: Fixed by https://github.com/acterglobal/a3/pull/1539/commits/0a6fcae7725f68ece46f80a92bcfccf852c8f2c0

File: app/lib/l10n/app_en.arb (hunk 3 above)
Comment (bitfriend) [reply app/lib/l10n/app_en.arb]: Fixed by https://github.com/acterglobal/a3/pull/1539/commits/0a6fcae7725f68ece46f80a92bcfccf852c8f2c0

**Pull request merged**